docs/ref/models/querysets.txt

```diff
@@ -2230,13 +2230,19 @@ instances, generally with one query::
     >>> objs[0].headline = 'This is entry 1'
     >>> objs[1].headline = 'This is entry 2'
     >>> Entry.objects.bulk_update(objs, ['headline'])
+    2
+
+.. versionchanged:: 4.0
+
+    This function earlier returned ``None``. Now, it returns the count of the
+     objects updated.
 
 :meth:`.QuerySet.update` is used to save the changes, so this is more efficient
 than iterating through the list of models and calling ``save()`` on each of
 them, but it has a few caveats:
 
-* You cannot update the model's primary key.
-* Each model's ``save()`` method isn't called, and the
+* You cannot update the object's primary key.
+* Each object's ``save()`` method isn't called, and the
```

Comment:
This change is not related with the ticket. Please revert it.

Reply:
i made this into a separate commit. Due to want me to create a new pull request for this?

Reply:
As far as I'm aware the current wording is correct. These notes are about model's primary key (model's field) and `save()` method.

Reply:
primary key is an attribute on the object of the model class correct? similarly with `save`.

Reply:
I think `model` is acceptable here. I understand why you may have been confused, but `object` is worse - too vague. Really you were trying to refer to a `model instance`, but the primary key and `.save()` method are defined by the model, not the instance.

Reply:
okay, i will drop the commit.